.github/workflows/ci-comment.yml

```diff
@@ -55,4 +55,25 @@ jobs:
               issue_number: issue_number,
               body: `The commit ${pr_sha} (as a parent of ${merge_sha}) contains errors. Please inspect the [Run Summary](${context.payload.workflow_run.html_url}) for details.`
             });
+            await octokit.request(`POST /repos/{owner}/{repo}/issues/{issue_number}/labels`, {
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: issue_number,
+              labels: [ 'waiting: response from author' ]
+            })
+
+      - name: On Success
+        uses: actions/github-script@v6
+        if: ${{ github.event.workflow_run.conclusion != 'failure' }}
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            let fs = require('fs');
+            let issue_number = Number(fs.readFileSync('./pr_number', 'utf8').trim());
+            await octokit.request(`DELETE /repos/{owner}/{repo}/issues/{issue_number}/labels/{name}`, {
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: issue_number,
+              name: 'waiting: response from author'
+            })
```

Comment:
As mentioned above, we should similarly use https://octokit.github.io/rest.js/v18#issues-remove-label for this.

Reply:
The problem is fetching the `issue_number`. I'm planning on making a few PRs to clear up some loose ends in the CI.